type string has no field or method Hex

I get this repo by run `go get github.com/edgexfoundry/device-sdk-go` , Then run 'make build', it raise the error:
```
CGO_ENABLED=0 go build -o example/cmd/device-simple/device-simple ./example/cmd/device-simple
# github.com/edgexfoundry/device-sdk-go/internal/cache
internal/cache/valuedescriptors.go:54:25: descriptor.Id.Hex undefined (type string has no field or method Hex)
internal/cache/valuedescriptors.go:59:34: descriptor.Id.Hex undefined (type string has no field or method Hex)
internal/cache/valuedescriptors.go:79:25: vd.Id.Hex undefined (type string has no field or method Hex)
internal/cache/valuedescriptors.go:90:16: vd.Id.Hex undefined (type string has no field or method Hex)
Makefile:13: recipe for target 'example/cmd/device-simple/device-simple' failed
make: *** [example/cmd/device-simple/device-simple] Error 2
```

--- a/internal/cache/valuedescriptors.go
+++ b/internal/cache/valuedescriptors.go
@@ -51,12 +51,12 @@ func (v *valueDescriptorCache) Add(descriptor models.ValueDescriptor) error {
 		return fmt.Errorf("value descriptor %s has already existed in cache", descriptor.Name)
 	}
 	v.vdMap[descriptor.Name] = descriptor
-	v.nameMap[descriptor.Id.Hex()] = descriptor.Name
+	v.nameMap[descriptor.Id] = descriptor.Name
 	return nil
 }
 
 func (v *valueDescriptorCache) Update(descriptor models.ValueDescriptor) error {
-	if err := v.Remove(descriptor.Id.Hex()); err != nil {
+	if err := v.Remove(descriptor.Id); err != nil {
 		return err
 	}
 	return v.Add(descriptor)
@@ -76,7 +76,7 @@ func (v *valueDescriptorCache) RemoveByName(name string) error {
 	if !ok {
 		return fmt.Errorf("value descriptor %s does not exist in cache", name)
 	}
-	delete(v.nameMap, vd.Id.Hex())
+	delete(v.nameMap, vd.Id)
 	delete(v.vdMap, name)
 	return nil
 }
@@ -87,7 +87,7 @@ func newValueDescriptorCache(descriptors []models.ValueDescriptor) ValueDescript
 	nameMap := make(map[string]string, defaultSize)
 	for _, vd := range descriptors {
 		vdMap[vd.Name] = vd
-		nameMap[vd.Id.Hex()] = vd.Name
+		nameMap[vd.Id] = vd.Name
 	}
 	vdc = &valueDescriptorCache{vdMap: vdMap, nameMap: nameMap}
 	return vdc

--- a/internal/clients/init_test.go
+++ b/internal/clients/init_test.go
@@ -31,7 +31,7 @@ func TestCheckServiceAvailableByPingWithTimeoutError(test *testing.T) {
 	var clientConfig = map[string]common.ClientInfo{common.ClientData: common.ClientInfo{Protocol: "http", Host: "www.google.com", Port: 81, Timeout: 3000}}
 	var config = common.Config{Clients: clientConfig}
 	common.CurrentConfig = &config
-	common.LoggingClient = logger.NewClient("test_service", false, common.CurrentConfig.Logging.File, "DEBUG")
+	common.LoggingClient = logger.NewClient("test_service", false, "./device-simple.log", "DEBUG")
 
 	err := checkServiceAvailableByPing(common.ClientData)
 

--- a/internal/controller/restfuncs_test.go
+++ b/internal/controller/restfuncs_test.go
@@ -43,7 +43,7 @@ func TestCallback(t *testing.T) {
 		{"Invalid id", http.MethodPut, `{"id":"5b9a4f9a64562a2f966fdb0b","type":"DEVICE"}`, http.StatusBadRequest},
 	}
 
-	lc := logger.NewClient("update_test", false, "", "DEBUG")
+	lc := logger.NewClient("update_test", false, "./device-simple.log", "DEBUG")
 	common.LoggingClient = lc
 	common.DeviceClient = &mock.DeviceClientMock{}
 	r := InitRestRoutes()

--- a/internal/provision/profiles.go
+++ b/internal/provision/profiles.go
@@ -158,7 +158,7 @@ func createDescriptor(name string, devObj models.DeviceObject) (*models.ValueDes
 		return nil, err
 	}
 
-	desc.Id = bson.ObjectIdHex(id)
+	desc.Id = id
 	common.LoggingClient.Debug(fmt.Sprintf("profiles: created Value Descriptor id: %s", id))
 
 	return desc, nil

